### ISSUE TITLE
Does the discriminator need pretraining?

### ISSUE THREAD

**Issue by ThibaultGROUEIX** [tianweiy/DMD2#74]
Hi @tianweiy ,

Awesome work on this project, super super impactful.
I am wondering why there is no pre-training for the GAN discriminator (unless I missed it). My understanding is that it would yield noisy gradients at the beginning of training if it is not initialize to distinguish between the fake and real distribution,

Best,
Thibault

**Comment by ThibaultGROUEIX** [comment]
Michael Gharbi asnwered separately:
1/ the GAN is pretty small so trains quickly
2/ at the beginning of training, the student generator also generates poor images and dfake is not tracking that distribution correctly either. 

personal TLDR, it might be good to run a few thousand steps of dfake + gan initialization, but in practice it seems unnecessary.
